### ISSUE TITLE
A1: R3 `/` Homepage - Update Form has no existence requirement

### ISSUE THREAD

**Issue by charasyn** [CISC-CMPE-327/Information#31]
Here is an excerpt from the homepage requirements:
- This page contains a form that a user can submit new tickets for sell. Fields: name, quantity, price, expiration date
- This page contains a form that a user can buy new tickets. Fields: name, quantity
- The ticket-selling form can be posted to /sell
- The ticket-buying form can be posted to /buy
- The ticket-update form can be posted to /update

For both buying and selling, we have a requirement that the page contains the form, but for ticket update, there is no requirement that there is a ticket update form. However, it still does require that we can post the form to `/update`.

How should we deal with this? Should we add an additional testcase ensuring that an update form exists?

**Comment by steven-hh-ding** [comment]
Yes intended discrepancy between the routing table and R3. R3 also misses a case that make sure the form exists. nice. please fork-and-pr.

**Comment by charasyn** [comment]
Hello Prof. Ding,
I don't see the discrepancy between routing table and R3.. in both it appears to show the correct methods for the `/`, `/logout`, `/buy`, `/sell`, and `/update` endpoints.
However the issue with the update form was accidentally fixed with #32 (I had meant only to include the fix for the `/update` endpoint, but I accidentally included the change to R3 - I'm still learning git 😄).
Based on this I believe this issue is resolved.
